docs/integrations/sources/linkedin-ads-migrations.md

```diff
@@ -1,5 +1,31 @@
 # LinkedIn Ads Migration Guide
 
+## Upgrading to 5.0.0
```

Comment:
the breaking change is about the change in primary key. It's not actually related to the migration to low-code

Reply:
agree with you 
PR with breaking changes has been created https://github.com/airbytehq/airbyte/pull/45353